### ISSUE TITLE
Screen reader announces "Okay" button in modals briefly when dismissing

### ISSUE THREAD

**Issue by eventualbuddha** [votingworks/bmd#985]
**Reproduction Steps**
1. Begin voting
2. Enable the screen reader by pressing `r`
3. Select a candidate in a 1-seat contest
4. Select another candidate in the same contest
5. You should see the modal telling you that you have to deselect the other one first
6. Press "Enter" to dismiss the modal

**Expected**
The current screen's default audio focus element should be read again.

**Actual**
The "Expected" does happen, but only after the "Okay" button is briefly announced as "O-" before being cut off when it starts announcing something else.

**Notes**
I had this same problem with navigation between contests with "Next" and "Back" and fixed it by triggering the screen reader only when an event didn't cause the location to change. We can't use that trick here because the location doesn't change, but maybe something similar would work.

_Originally posted by @eventualbuddha in https://github.com/votingworks/bmd/pull/966#issuecomment-567335172_
